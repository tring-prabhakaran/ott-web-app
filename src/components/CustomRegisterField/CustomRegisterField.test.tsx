import React from 'react';
import { render } from '@testing-library/react';

import { CustomRegisterField } from './CustomRegisterField';

describe('<CustomRegisterField>', () => {
  test('renders and matches snapshot <Checkbox>', () => {
    const { container } = render(<CustomRegisterField type="checkbox" label="label" name="name" value="value" onChange={vi.fn()} />);

    expect(container).toMatchSnapshot();
  });

  test('renders and matches snapshot <TextField>', () => {
    const { container } = render(<CustomRegisterField type="input" label="label" name="name" value="value" onChange={vi.fn()} />);

    expect(container).toMatchSnapshot();
  });

  test('renders and matches snapshot <Radio>', () => {
    const { container } = render(<CustomRegisterField type="radio" label="label" name="name" value="value" onChange={vi.fn()} />);

    expect(container).toMatchSnapshot();
  });

  test('renders and matches snapshot <Dropdown type="select">', () => {
    const { container } = render(<CustomRegisterField type="select" label="label" name="name" value="value" onChange={vi.fn()} />);

    expect(container).toMatchSnapshot();
  });

  test('renders and matches snapshot <Dropdown type="country">', () => {
    const { container } = render(<CustomRegisterField type="country" label="label" name="name" value="value" onChange={vi.fn()} />);

    expect(container).toMatchSnapshot();
  });

  test('renders and matches snapshot <Dropdown type="us_state">', () => {
    const { container } = render(<CustomRegisterField type="us_state" label="label" name="name" value="value" onChange={vi.fn()} />);

    expect(container).toMatchSnapshot();
  });

  test('renders and matches snapshot <Dropdown type="datepicker">', () => {
    const { container } = render(<CustomRegisterField type="datepicker" label="label" name="name" value="value" onChange={vi.fn()} />);

    expect(container).toMatchSnapshot();
  });

  test('renders and matches snapshot <Dropdown type="randomstring">', () => {
    // @ts-ignore
    const { container } = render(<CustomRegisterField type="randomstring" label="label" name="name" value="value" onChange={vi.fn()} />);

    expect(container).toMatchSnapshot();
  });
});
