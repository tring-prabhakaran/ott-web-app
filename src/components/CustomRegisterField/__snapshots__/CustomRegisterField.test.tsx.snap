// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<CustomRegisterField> > renders and matches snapshot <Checkbox> 1`] = `
<div>
  <div
    class="_checkbox_531f07"
  >
    <div
      class="_row_531f07"
    >
      <input
        id="check-box_1235_name"
        name="name"
        type="checkbox"
        value=""
      />
      <label
        for="check-box_1235_name"
      >
        label
      </label>
    </div>
  </div>
</div>
`;

exports[`<CustomRegisterField> > renders and matches snapshot <Dropdown type="country"> 1`] = `
<div>
  <div
    class="_container_bb73c6 _fullWidth_bb73c6 _medium_bb73c6"
  >
    <label
      class="_label_bb73c6"
      for="1235"
    >
      label
      <span>
        optional
      </span>
    </label>
    <div
      class="_dropdown_bb73c6 _fullWidth_bb73c6"
    >
      <select
        aria-required="false"
        class="_select_bb73c6"
        id="1235"
        name="name"
      >
        <option
          class="_option_bb73c6"
          value="af"
        >
          country:af
        </option>
        <option
          class="_option_bb73c6"
          value="ax"
        >
          country:ax
        </option>
        <option
          class="_option_bb73c6"
          value="al"
        >
          country:al
        </option>
        <option
          class="_option_bb73c6"
          value="dz"
        >
          country:dz
        </option>
        <option
          class="_option_bb73c6"
          value="as"
        >
          country:as
        </option>
        <option
          class="_option_bb73c6"
          value="ad"
        >
          country:ad
        </option>
        <option
          class="_option_bb73c6"
          value="ao"
        >
          country:ao
        </option>
        <option
          class="_option_bb73c6"
          value="ai"
        >
          country:ai
        </option>
        <option
          class="_option_bb73c6"
          value="aq"
        >
          country:aq
        </option>
        <option
          class="_option_bb73c6"
          value="ag"
        >
          country:ag
        </option>
        <option
          class="_option_bb73c6"
          value="ar"
        >
          country:ar
        </option>
        <option
          class="_option_bb73c6"
          value="am"
        >
          country:am
        </option>
        <option
          class="_option_bb73c6"
          value="aw"
        >
          country:aw
        </option>
        <option
          class="_option_bb73c6"
          value="au"
        >
          country:au
        </option>
        <option
          class="_option_bb73c6"
          value="at"
        >
          country:at
        </option>
        <option
          class="_option_bb73c6"
          value="az"
        >
          country:az
        </option>
        <option
          class="_option_bb73c6"
          value="bs"
        >
          country:bs
        </option>
        <option
          class="_option_bb73c6"
          value="bh"
        >
          country:bh
        </option>
        <option
          class="_option_bb73c6"
          value="bd"
        >
          country:bd
        </option>
        <option
          class="_option_bb73c6"
          value="bb"
        >
          country:bb
        </option>
        <option
          class="_option_bb73c6"
          value="by"
        >
          country:by
        </option>
        <option
          class="_option_bb73c6"
          value="be"
        >
          country:be
        </option>
        <option
          class="_option_bb73c6"
          value="bz"
        >
          country:bz
        </option>
        <option
          class="_option_bb73c6"
          value="bj"
        >
          country:bj
        </option>
        <option
          class="_option_bb73c6"
          value="bm"
        >
          country:bm
        </option>
        <option
          class="_option_bb73c6"
          value="bt"
        >
          country:bt
        </option>
        <option
          class="_option_bb73c6"
          value="bo"
        >
          country:bo
        </option>
        <option
          class="_option_bb73c6"
          value="bq"
        >
          country:bq
        </option>
        <option
          class="_option_bb73c6"
          value="ba"
        >
          country:ba
        </option>
        <option
          class="_option_bb73c6"
          value="bw"
        >
          country:bw
        </option>
        <option
          class="_option_bb73c6"
          value="bv"
        >
          country:bv
        </option>
        <option
          class="_option_bb73c6"
          value="br"
        >
          country:br
        </option>
        <option
          class="_option_bb73c6"
          value="io"
        >
          country:io
        </option>
        <option
          class="_option_bb73c6"
          value="bn"
        >
          country:bn
        </option>
        <option
          class="_option_bb73c6"
          value="bg"
        >
          country:bg
        </option>
        <option
          class="_option_bb73c6"
          value="bf"
        >
          country:bf
        </option>
        <option
          class="_option_bb73c6"
          value="bi"
        >
          country:bi
        </option>
        <option
          class="_option_bb73c6"
          value="kh"
        >
          country:kh
        </option>
        <option
          class="_option_bb73c6"
          value="cm"
        >
          country:cm
        </option>
        <option
          class="_option_bb73c6"
          value="ca"
        >
          country:ca
        </option>
        <option
          class="_option_bb73c6"
          value="cv"
        >
          country:cv
        </option>
        <option
          class="_option_bb73c6"
          value="ky"
        >
          country:ky
        </option>
        <option
          class="_option_bb73c6"
          value="cf"
        >
          country:cf
        </option>
        <option
          class="_option_bb73c6"
          value="td"
        >
          country:td
        </option>
        <option
          class="_option_bb73c6"
          value="cl"
        >
          country:cl
        </option>
        <option
          class="_option_bb73c6"
          value="cn"
        >
          country:cn
        </option>
        <option
          class="_option_bb73c6"
          value="cx"
        >
          country:cx
        </option>
        <option
          class="_option_bb73c6"
          value="cc"
        >
          country:cc
        </option>
        <option
          class="_option_bb73c6"
          value="co"
        >
          country:co
        </option>
        <option
          class="_option_bb73c6"
          value="km"
        >
          country:km
        </option>
        <option
          class="_option_bb73c6"
          value="cg"
        >
          country:cg
        </option>
        <option
          class="_option_bb73c6"
          value="cd"
        >
          country:cd
        </option>
        <option
          class="_option_bb73c6"
          value="ck"
        >
          country:ck
        </option>
        <option
          class="_option_bb73c6"
          value="cr"
        >
          country:cr
        </option>
        <option
          class="_option_bb73c6"
          value="ci"
        >
          country:ci
        </option>
        <option
          class="_option_bb73c6"
          value="hr"
        >
          country:hr
        </option>
        <option
          class="_option_bb73c6"
          value="cu"
        >
          country:cu
        </option>
        <option
          class="_option_bb73c6"
          value="cw"
        >
          country:cw
        </option>
        <option
          class="_option_bb73c6"
          value="cy"
        >
          country:cy
        </option>
        <option
          class="_option_bb73c6"
          value="cz"
        >
          country:cz
        </option>
        <option
          class="_option_bb73c6"
          value="dk"
        >
          country:dk
        </option>
        <option
          class="_option_bb73c6"
          value="dj"
        >
          country:dj
        </option>
        <option
          class="_option_bb73c6"
          value="dm"
        >
          country:dm
        </option>
        <option
          class="_option_bb73c6"
          value="do"
        >
          country:do
        </option>
        <option
          class="_option_bb73c6"
          value="ec"
        >
          country:ec
        </option>
        <option
          class="_option_bb73c6"
          value="eg"
        >
          country:eg
        </option>
        <option
          class="_option_bb73c6"
          value="sv"
        >
          country:sv
        </option>
        <option
          class="_option_bb73c6"
          value="gq"
        >
          country:gq
        </option>
        <option
          class="_option_bb73c6"
          value="er"
        >
          country:er
        </option>
        <option
          class="_option_bb73c6"
          value="ee"
        >
          country:ee
        </option>
        <option
          class="_option_bb73c6"
          value="et"
        >
          country:et
        </option>
        <option
          class="_option_bb73c6"
          value="fk"
        >
          country:fk
        </option>
        <option
          class="_option_bb73c6"
          value="fo"
        >
          country:fo
        </option>
        <option
          class="_option_bb73c6"
          value="fj"
        >
          country:fj
        </option>
        <option
          class="_option_bb73c6"
          value="fi"
        >
          country:fi
        </option>
        <option
          class="_option_bb73c6"
          value="fr"
        >
          country:fr
        </option>
        <option
          class="_option_bb73c6"
          value="gf"
        >
          country:gf
        </option>
        <option
          class="_option_bb73c6"
          value="pf"
        >
          country:pf
        </option>
        <option
          class="_option_bb73c6"
          value="tf"
        >
          country:tf
        </option>
        <option
          class="_option_bb73c6"
          value="ga"
        >
          country:ga
        </option>
        <option
          class="_option_bb73c6"
          value="gm"
        >
          country:gm
        </option>
        <option
          class="_option_bb73c6"
          value="ge"
        >
          country:ge
        </option>
        <option
          class="_option_bb73c6"
          value="de"
        >
          country:de
        </option>
        <option
          class="_option_bb73c6"
          value="gh"
        >
          country:gh
        </option>
        <option
          class="_option_bb73c6"
          value="gi"
        >
          country:gi
        </option>
        <option
          class="_option_bb73c6"
          value="gr"
        >
          country:gr
        </option>
        <option
          class="_option_bb73c6"
          value="gl"
        >
          country:gl
        </option>
        <option
          class="_option_bb73c6"
          value="gd"
        >
          country:gd
        </option>
        <option
          class="_option_bb73c6"
          value="gp"
        >
          country:gp
        </option>
        <option
          class="_option_bb73c6"
          value="gu"
        >
          country:gu
        </option>
        <option
          class="_option_bb73c6"
          value="gt"
        >
          country:gt
        </option>
        <option
          class="_option_bb73c6"
          value="gg"
        >
          country:gg
        </option>
        <option
          class="_option_bb73c6"
          value="gn"
        >
          country:gn
        </option>
        <option
          class="_option_bb73c6"
          value="gw"
        >
          country:gw
        </option>
        <option
          class="_option_bb73c6"
          value="gy"
        >
          country:gy
        </option>
        <option
          class="_option_bb73c6"
          value="ht"
        >
          country:ht
        </option>
        <option
          class="_option_bb73c6"
          value="hm"
        >
          country:hm
        </option>
        <option
          class="_option_bb73c6"
          value="va"
        >
          country:va
        </option>
        <option
          class="_option_bb73c6"
          value="hn"
        >
          country:hn
        </option>
        <option
          class="_option_bb73c6"
          value="hk"
        >
          country:hk
        </option>
        <option
          class="_option_bb73c6"
          value="hu"
        >
          country:hu
        </option>
        <option
          class="_option_bb73c6"
          value="is"
        >
          country:is
        </option>
        <option
          class="_option_bb73c6"
          value="in"
        >
          country:in
        </option>
        <option
          class="_option_bb73c6"
          value="id"
        >
          country:id
        </option>
        <option
          class="_option_bb73c6"
          value="ir"
        >
          country:ir
        </option>
        <option
          class="_option_bb73c6"
          value="iq"
        >
          country:iq
        </option>
        <option
          class="_option_bb73c6"
          value="ie"
        >
          country:ie
        </option>
        <option
          class="_option_bb73c6"
          value="im"
        >
          country:im
        </option>
        <option
          class="_option_bb73c6"
          value="il"
        >
          country:il
        </option>
        <option
          class="_option_bb73c6"
          value="it"
        >
          country:it
        </option>
        <option
          class="_option_bb73c6"
          value="jm"
        >
          country:jm
        </option>
        <option
          class="_option_bb73c6"
          value="jp"
        >
          country:jp
        </option>
        <option
          class="_option_bb73c6"
          value="je"
        >
          country:je
        </option>
        <option
          class="_option_bb73c6"
          value="jo"
        >
          country:jo
        </option>
        <option
          class="_option_bb73c6"
          value="kz"
        >
          country:kz
        </option>
        <option
          class="_option_bb73c6"
          value="ke"
        >
          country:ke
        </option>
        <option
          class="_option_bb73c6"
          value="ki"
        >
          country:ki
        </option>
        <option
          class="_option_bb73c6"
          value="kp"
        >
          country:kp
        </option>
        <option
          class="_option_bb73c6"
          value="kr"
        >
          country:kr
        </option>
        <option
          class="_option_bb73c6"
          value="kw"
        >
          country:kw
        </option>
        <option
          class="_option_bb73c6"
          value="kg"
        >
          country:kg
        </option>
        <option
          class="_option_bb73c6"
          value="la"
        >
          country:la
        </option>
        <option
          class="_option_bb73c6"
          value="lv"
        >
          country:lv
        </option>
        <option
          class="_option_bb73c6"
          value="lb"
        >
          country:lb
        </option>
        <option
          class="_option_bb73c6"
          value="ls"
        >
          country:ls
        </option>
        <option
          class="_option_bb73c6"
          value="lr"
        >
          country:lr
        </option>
        <option
          class="_option_bb73c6"
          value="ly"
        >
          country:ly
        </option>
        <option
          class="_option_bb73c6"
          value="li"
        >
          country:li
        </option>
        <option
          class="_option_bb73c6"
          value="lt"
        >
          country:lt
        </option>
        <option
          class="_option_bb73c6"
          value="lu"
        >
          country:lu
        </option>
        <option
          class="_option_bb73c6"
          value="mo"
        >
          country:mo
        </option>
        <option
          class="_option_bb73c6"
          value="mk"
        >
          country:mk
        </option>
        <option
          class="_option_bb73c6"
          value="mg"
        >
          country:mg
        </option>
        <option
          class="_option_bb73c6"
          value="mw"
        >
          country:mw
        </option>
        <option
          class="_option_bb73c6"
          value="my"
        >
          country:my
        </option>
        <option
          class="_option_bb73c6"
          value="mv"
        >
          country:mv
        </option>
        <option
          class="_option_bb73c6"
          value="ml"
        >
          country:ml
        </option>
        <option
          class="_option_bb73c6"
          value="mt"
        >
          country:mt
        </option>
        <option
          class="_option_bb73c6"
          value="mh"
        >
          country:mh
        </option>
        <option
          class="_option_bb73c6"
          value="mq"
        >
          country:mq
        </option>
        <option
          class="_option_bb73c6"
          value="mr"
        >
          country:mr
        </option>
        <option
          class="_option_bb73c6"
          value="mu"
        >
          country:mu
        </option>
        <option
          class="_option_bb73c6"
          value="yt"
        >
          country:yt
        </option>
        <option
          class="_option_bb73c6"
          value="mx"
        >
          country:mx
        </option>
        <option
          class="_option_bb73c6"
          value="fm"
        >
          country:fm
        </option>
        <option
          class="_option_bb73c6"
          value="md"
        >
          country:md
        </option>
        <option
          class="_option_bb73c6"
          value="mc"
        >
          country:mc
        </option>
        <option
          class="_option_bb73c6"
          value="mn"
        >
          country:mn
        </option>
        <option
          class="_option_bb73c6"
          value="me"
        >
          country:me
        </option>
        <option
          class="_option_bb73c6"
          value="ms"
        >
          country:ms
        </option>
        <option
          class="_option_bb73c6"
          value="ma"
        >
          country:ma
        </option>
        <option
          class="_option_bb73c6"
          value="mz"
        >
          country:mz
        </option>
        <option
          class="_option_bb73c6"
          value="mm"
        >
          country:mm
        </option>
        <option
          class="_option_bb73c6"
          value="na"
        >
          country:na
        </option>
        <option
          class="_option_bb73c6"
          value="nr"
        >
          country:nr
        </option>
        <option
          class="_option_bb73c6"
          value="np"
        >
          country:np
        </option>
        <option
          class="_option_bb73c6"
          value="nl"
        >
          country:nl
        </option>
        <option
          class="_option_bb73c6"
          value="nc"
        >
          country:nc
        </option>
        <option
          class="_option_bb73c6"
          value="nz"
        >
          country:nz
        </option>
        <option
          class="_option_bb73c6"
          value="ni"
        >
          country:ni
        </option>
        <option
          class="_option_bb73c6"
          value="ne"
        >
          country:ne
        </option>
        <option
          class="_option_bb73c6"
          value="ng"
        >
          country:ng
        </option>
        <option
          class="_option_bb73c6"
          value="nu"
        >
          country:nu
        </option>
        <option
          class="_option_bb73c6"
          value="nf"
        >
          country:nf
        </option>
        <option
          class="_option_bb73c6"
          value="mp"
        >
          country:mp
        </option>
        <option
          class="_option_bb73c6"
          value="no"
        >
          country:no
        </option>
        <option
          class="_option_bb73c6"
          value="om"
        >
          country:om
        </option>
        <option
          class="_option_bb73c6"
          value="pk"
        >
          country:pk
        </option>
        <option
          class="_option_bb73c6"
          value="pw"
        >
          country:pw
        </option>
        <option
          class="_option_bb73c6"
          value="ps"
        >
          country:ps
        </option>
        <option
          class="_option_bb73c6"
          value="pa"
        >
          country:pa
        </option>
        <option
          class="_option_bb73c6"
          value="pg"
        >
          country:pg
        </option>
        <option
          class="_option_bb73c6"
          value="py"
        >
          country:py
        </option>
        <option
          class="_option_bb73c6"
          value="pe"
        >
          country:pe
        </option>
        <option
          class="_option_bb73c6"
          value="ph"
        >
          country:ph
        </option>
        <option
          class="_option_bb73c6"
          value="pn"
        >
          country:pn
        </option>
        <option
          class="_option_bb73c6"
          value="pl"
        >
          country:pl
        </option>
        <option
          class="_option_bb73c6"
          value="pt"
        >
          country:pt
        </option>
        <option
          class="_option_bb73c6"
          value="pr"
        >
          country:pr
        </option>
        <option
          class="_option_bb73c6"
          value="qa"
        >
          country:qa
        </option>
        <option
          class="_option_bb73c6"
          value="re"
        >
          country:re
        </option>
        <option
          class="_option_bb73c6"
          value="ro"
        >
          country:ro
        </option>
        <option
          class="_option_bb73c6"
          value="ru"
        >
          country:ru
        </option>
        <option
          class="_option_bb73c6"
          value="rw"
        >
          country:rw
        </option>
        <option
          class="_option_bb73c6"
          value="bl"
        >
          country:bl
        </option>
        <option
          class="_option_bb73c6"
          value="sh"
        >
          country:sh
        </option>
        <option
          class="_option_bb73c6"
          value="kn"
        >
          country:kn
        </option>
        <option
          class="_option_bb73c6"
          value="lc"
        >
          country:lc
        </option>
        <option
          class="_option_bb73c6"
          value="mf"
        >
          country:mf
        </option>
        <option
          class="_option_bb73c6"
          value="pm"
        >
          country:pm
        </option>
        <option
          class="_option_bb73c6"
          value="vc"
        >
          country:vc
        </option>
        <option
          class="_option_bb73c6"
          value="ws"
        >
          country:ws
        </option>
        <option
          class="_option_bb73c6"
          value="sm"
        >
          country:sm
        </option>
        <option
          class="_option_bb73c6"
          value="st"
        >
          country:st
        </option>
        <option
          class="_option_bb73c6"
          value="sa"
        >
          country:sa
        </option>
        <option
          class="_option_bb73c6"
          value="sn"
        >
          country:sn
        </option>
        <option
          class="_option_bb73c6"
          value="rs"
        >
          country:rs
        </option>
        <option
          class="_option_bb73c6"
          value="sc"
        >
          country:sc
        </option>
        <option
          class="_option_bb73c6"
          value="sl"
        >
          country:sl
        </option>
        <option
          class="_option_bb73c6"
          value="sg"
        >
          country:sg
        </option>
        <option
          class="_option_bb73c6"
          value="sx"
        >
          country:sx
        </option>
        <option
          class="_option_bb73c6"
          value="sk"
        >
          country:sk
        </option>
        <option
          class="_option_bb73c6"
          value="si"
        >
          country:si
        </option>
        <option
          class="_option_bb73c6"
          value="sb"
        >
          country:sb
        </option>
        <option
          class="_option_bb73c6"
          value="so"
        >
          country:so
        </option>
        <option
          class="_option_bb73c6"
          value="za"
        >
          country:za
        </option>
        <option
          class="_option_bb73c6"
          value="gs"
        >
          country:gs
        </option>
        <option
          class="_option_bb73c6"
          value="es"
        >
          country:es
        </option>
        <option
          class="_option_bb73c6"
          value="lk"
        >
          country:lk
        </option>
        <option
          class="_option_bb73c6"
          value="sd"
        >
          country:sd
        </option>
        <option
          class="_option_bb73c6"
          value="sr"
        >
          country:sr
        </option>
        <option
          class="_option_bb73c6"
          value="sj"
        >
          country:sj
        </option>
        <option
          class="_option_bb73c6"
          value="sz"
        >
          country:sz
        </option>
        <option
          class="_option_bb73c6"
          value="se"
        >
          country:se
        </option>
        <option
          class="_option_bb73c6"
          value="ch"
        >
          country:ch
        </option>
        <option
          class="_option_bb73c6"
          value="sy"
        >
          country:sy
        </option>
        <option
          class="_option_bb73c6"
          value="tw"
        >
          country:tw
        </option>
        <option
          class="_option_bb73c6"
          value="tj"
        >
          country:tj
        </option>
        <option
          class="_option_bb73c6"
          value="tz"
        >
          country:tz
        </option>
        <option
          class="_option_bb73c6"
          value="th"
        >
          country:th
        </option>
        <option
          class="_option_bb73c6"
          value="tl"
        >
          country:tl
        </option>
        <option
          class="_option_bb73c6"
          value="tg"
        >
          country:tg
        </option>
        <option
          class="_option_bb73c6"
          value="tk"
        >
          country:tk
        </option>
        <option
          class="_option_bb73c6"
          value="to"
        >
          country:to
        </option>
        <option
          class="_option_bb73c6"
          value="tt"
        >
          country:tt
        </option>
        <option
          class="_option_bb73c6"
          value="tn"
        >
          country:tn
        </option>
        <option
          class="_option_bb73c6"
          value="tr"
        >
          country:tr
        </option>
        <option
          class="_option_bb73c6"
          value="tm"
        >
          country:tm
        </option>
        <option
          class="_option_bb73c6"
          value="tc"
        >
          country:tc
        </option>
        <option
          class="_option_bb73c6"
          value="tv"
        >
          country:tv
        </option>
        <option
          class="_option_bb73c6"
          value="ug"
        >
          country:ug
        </option>
        <option
          class="_option_bb73c6"
          value="ua"
        >
          country:ua
        </option>
        <option
          class="_option_bb73c6"
          value="ae"
        >
          country:ae
        </option>
        <option
          class="_option_bb73c6"
          value="gb"
        >
          country:gb
        </option>
        <option
          class="_option_bb73c6"
          value="us"
        >
          country:us
        </option>
        <option
          class="_option_bb73c6"
          value="um"
        >
          country:um
        </option>
        <option
          class="_option_bb73c6"
          value="uy"
        >
          country:uy
        </option>
        <option
          class="_option_bb73c6"
          value="uz"
        >
          country:uz
        </option>
        <option
          class="_option_bb73c6"
          value="vu"
        >
          country:vu
        </option>
        <option
          class="_option_bb73c6"
          value="ve"
        >
          country:ve
        </option>
        <option
          class="_option_bb73c6"
          value="vn"
        >
          country:vn
        </option>
        <option
          class="_option_bb73c6"
          value="vg"
        >
          country:vg
        </option>
        <option
          class="_option_bb73c6"
          value="vi"
        >
          country:vi
        </option>
        <option
          class="_option_bb73c6"
          value="wf"
        >
          country:wf
        </option>
        <option
          class="_option_bb73c6"
          value="eh"
        >
          country:eh
        </option>
        <option
          class="_option_bb73c6"
          value="ye"
        >
          country:ye
        </option>
        <option
          class="_option_bb73c6"
          value="zm"
        >
          country:zm
        </option>
        <option
          class="_option_bb73c6"
          value="zw"
        >
          country:zw
        </option>
      </select>
    </div>
  </div>
</div>
`;

exports[`<CustomRegisterField> > renders and matches snapshot <Dropdown type="datepicker"> 1`] = `
<div>
  <div
    class="_dateField_eb6974"
  >
    <label
      class="_label_eb6974"
      for="text-field_1235_name"
    >
      label
      <span>
        optional
      </span>
    </label>
    <div
      class="_container_eb6974"
    >
      <input
        class="_hiddenInput_eb6974"
        id="text-field_1235_name"
        name="name"
      />
      <input
        class="_input_eb6974"
        id="text-field_1235_name-date"
        maxlength="2"
        name="date"
        placeholder="dd"
        type="number"
        value=""
      />
       / 
      <input
        class="_input_eb6974"
        id="text-field_1235_name-month"
        maxlength="2"
        name="month"
        placeholder="mm"
        type="number"
        value=""
      />
       / 
      <input
        class="_input_eb6974"
        id="text-field_1235_name-year"
        maxlength="4"
        name="year"
        placeholder="yyyy"
        type="number"
        value=""
      />
    </div>
  </div>
</div>
`;

exports[`<CustomRegisterField> > renders and matches snapshot <Dropdown type="randomstring"> 1`] = `
<div>
  <div
    class="_checkbox_531f07"
  >
    <div
      class="_row_531f07"
    >
      <input
        id="check-box_1235_name"
        name="name"
        type="checkbox"
        value=""
      />
      <label
        for="check-box_1235_name"
      >
        label
      </label>
    </div>
  </div>
</div>
`;

exports[`<CustomRegisterField> > renders and matches snapshot <Dropdown type="select"> 1`] = `
<div>
  <div
    class="_container_bb73c6 _fullWidth_bb73c6 _medium_bb73c6"
  >
    <label
      class="_label_bb73c6"
      for="1235"
    >
      label
      <span>
        optional
      </span>
    </label>
    <div
      class="_dropdown_bb73c6 _fullWidth_bb73c6"
    >
      <select
        aria-required="false"
        class="_select_bb73c6"
        id="1235"
        name="name"
      />
    </div>
  </div>
</div>
`;

exports[`<CustomRegisterField> > renders and matches snapshot <Dropdown type="us_state"> 1`] = `
<div>
  <div
    class="_container_bb73c6 _fullWidth_bb73c6 _medium_bb73c6"
  >
    <label
      class="_label_bb73c6"
      for="1235"
    >
      label
      <span>
        optional
      </span>
    </label>
    <div
      class="_dropdown_bb73c6 _fullWidth_bb73c6"
    >
      <select
        aria-required="false"
        class="_select_bb73c6"
        id="1235"
        name="name"
      >
        <option
          class="_option_bb73c6"
          value="al"
        >
          us_state:al
        </option>
        <option
          class="_option_bb73c6"
          value="ak"
        >
          us_state:ak
        </option>
        <option
          class="_option_bb73c6"
          value="az"
        >
          us_state:az
        </option>
        <option
          class="_option_bb73c6"
          value="ar"
        >
          us_state:ar
        </option>
        <option
          class="_option_bb73c6"
          value="ca"
        >
          us_state:ca
        </option>
        <option
          class="_option_bb73c6"
          value="co"
        >
          us_state:co
        </option>
        <option
          class="_option_bb73c6"
          value="ct"
        >
          us_state:ct
        </option>
        <option
          class="_option_bb73c6"
          value="de"
        >
          us_state:de
        </option>
        <option
          class="_option_bb73c6"
          value="dc"
        >
          us_state:dc
        </option>
        <option
          class="_option_bb73c6"
          value="fl"
        >
          us_state:fl
        </option>
        <option
          class="_option_bb73c6"
          value="ga"
        >
          us_state:ga
        </option>
        <option
          class="_option_bb73c6"
          value="hi"
        >
          us_state:hi
        </option>
        <option
          class="_option_bb73c6"
          value="id"
        >
          us_state:id
        </option>
        <option
          class="_option_bb73c6"
          value="il"
        >
          us_state:il
        </option>
        <option
          class="_option_bb73c6"
          value="in"
        >
          us_state:in
        </option>
        <option
          class="_option_bb73c6"
          value="ia"
        >
          us_state:ia
        </option>
        <option
          class="_option_bb73c6"
          value="ks"
        >
          us_state:ks
        </option>
        <option
          class="_option_bb73c6"
          value="ky"
        >
          us_state:ky
        </option>
        <option
          class="_option_bb73c6"
          value="la"
        >
          us_state:la
        </option>
        <option
          class="_option_bb73c6"
          value="me"
        >
          us_state:me
        </option>
        <option
          class="_option_bb73c6"
          value="md"
        >
          us_state:md
        </option>
        <option
          class="_option_bb73c6"
          value="ma"
        >
          us_state:ma
        </option>
        <option
          class="_option_bb73c6"
          value="mi"
        >
          us_state:mi
        </option>
        <option
          class="_option_bb73c6"
          value="mn"
        >
          us_state:mn
        </option>
        <option
          class="_option_bb73c6"
          value="ms"
        >
          us_state:ms
        </option>
        <option
          class="_option_bb73c6"
          value="mo"
        >
          us_state:mo
        </option>
        <option
          class="_option_bb73c6"
          value="mt"
        >
          us_state:mt
        </option>
        <option
          class="_option_bb73c6"
          value="ne"
        >
          us_state:ne
        </option>
        <option
          class="_option_bb73c6"
          value="nv"
        >
          us_state:nv
        </option>
        <option
          class="_option_bb73c6"
          value="nh"
        >
          us_state:nh
        </option>
        <option
          class="_option_bb73c6"
          value="nj"
        >
          us_state:nj
        </option>
        <option
          class="_option_bb73c6"
          value="nm"
        >
          us_state:nm
        </option>
        <option
          class="_option_bb73c6"
          value="ny"
        >
          us_state:ny
        </option>
        <option
          class="_option_bb73c6"
          value="nc"
        >
          us_state:nc
        </option>
        <option
          class="_option_bb73c6"
          value="nd"
        >
          us_state:nd
        </option>
        <option
          class="_option_bb73c6"
          value="oh"
        >
          us_state:oh
        </option>
        <option
          class="_option_bb73c6"
          value="ok"
        >
          us_state:ok
        </option>
        <option
          class="_option_bb73c6"
          value="or"
        >
          us_state:or
        </option>
        <option
          class="_option_bb73c6"
          value="pa"
        >
          us_state:pa
        </option>
        <option
          class="_option_bb73c6"
          value="ri"
        >
          us_state:ri
        </option>
        <option
          class="_option_bb73c6"
          value="sc"
        >
          us_state:sc
        </option>
        <option
          class="_option_bb73c6"
          value="sd"
        >
          us_state:sd
        </option>
        <option
          class="_option_bb73c6"
          value="tn"
        >
          us_state:tn
        </option>
        <option
          class="_option_bb73c6"
          value="tx"
        >
          us_state:tx
        </option>
        <option
          class="_option_bb73c6"
          value="ut"
        >
          us_state:ut
        </option>
        <option
          class="_option_bb73c6"
          value="vt"
        >
          us_state:vt
        </option>
        <option
          class="_option_bb73c6"
          value="va"
        >
          us_state:va
        </option>
        <option
          class="_option_bb73c6"
          value="wa"
        >
          us_state:wa
        </option>
        <option
          class="_option_bb73c6"
          value="wv"
        >
          us_state:wv
        </option>
        <option
          class="_option_bb73c6"
          value="wi"
        >
          us_state:wi
        </option>
        <option
          class="_option_bb73c6"
          value="wy"
        >
          us_state:wy
        </option>
      </select>
    </div>
  </div>
</div>
`;

exports[`<CustomRegisterField> > renders and matches snapshot <Radio> 1`] = `
<div>
  <div>
    <div
      class="_header_d50e6e"
    >
      label
      <span>
        optional
      </span>
    </div>
  </div>
</div>
`;

exports[`<CustomRegisterField> > renders and matches snapshot <TextField> 1`] = `
<div>
  <div
    class="_textField_e16c1b"
  >
    <label
      class="_label_e16c1b"
      for="text-field_1235_name"
    >
      label
      <span>
        optional
      </span>
    </label>
    <div
      class="_container_e16c1b"
    >
      <input
        class="_input_e16c1b"
        id="text-field_1235_name"
        name="name"
        type="text"
        value="value"
      />
    </div>
  </div>
</div>
`;
